group/templates/group/groupDetail.html

```diff
@@ -102,13 +108,21 @@ <h4 class="panel-title">
                 {% for running_contest in rc_list %}
```

Comment:
Could you use full name? Or it's very likely we forget the meaning after a while.